src/staking-router-modules/contracts/staking-router/staking-router-fetch.service.ts

```diff
@@ -42,29 +46,21 @@ export class StakingRouterFetchService {
 
     const transformedModules = await Promise.all(
       modules.map(async (stakingModule) => {
-        // const isActive = await contract.getStakingModuleIsActive(stakingModule.id, { blockTag } as any);
-        // until the end of voting work with old version of Node Operator Registry
-        // this version doesnt correspond to IStakingModule interface
-        // currently skip this section
+        // TODO: what is the diff between status and active ?
```

Comment:
Status can be one of 3: Active, DepositsPaused, Stopped. Method `getStakingModuleIsActive` just checks that status === 'Active'

Reply:
storage of status should be enough